README.md

```diff
@@ -34,6 +35,20 @@ to point to any subdirectory of the `machines` directory.
 
 See [machines/README.md](machines/README.md) for more documentation.
 
+## Execution Platforms
+
+HammerBlade runs on a variety of platforms: AWS F1 (natively),
+Synopsys VCS (simulation), and others. Each platform requires a
+different runtime library, hardware files, and build commands.
+We abstract these differences in the makefiles. 
+
+To switch platforms, set the variable `BSG_PLATFORM` in
+[platform.mk](platform.mk). Available platforms are subdirectories in
+[libraries/platforms](libraries/platforms). For the most part, users
+do not need to worry about what platform they are running on;
+`platform.mk` will automatically deduce the correct platform from
+the environment
+
 # Quick-Start
 
 The simplest way to use this project is to clone it's meta-project: [BSG Bladerunner](https://github.com/bespoke-silicon-group/bsg_bladerunner/). 
```

Comment:
"its meta-project"

Reply:
I was asking for a code review, not a grammar check..

(JUST KIDDING, I'm glad you found this.)

Reply:
Fixed

Reply:
:)